<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="startImage_Label">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>100</y>
        <width>400</width>
        <height>350</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/image/diy.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="title_Label">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>20</y>
        <width>371</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 36pt &quot;Agency FB&quot;;
color: rgb(255, 170, 0);</string>
      </property>
      <property name="text">
       <string>       电子贺卡</string>
      </property>
     </widget>
     <widget class="QPushButton" name="start_Btn">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>460</y>
        <width>141</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>现在开始！</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="page_2_previous_Btn">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>上一步</string>
      </property>
     </widget>
     <widget class="QPushButton" name="page_2_next_Btn">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>400</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>下一步</string>
      </property>
     </widget>
     <widget class="QLabel" name="E_Card_BackgroundImage_Label">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>630</width>
        <height>460</height>
       </rect>
      </property>
      <property name="text">
       <string>贺卡背景</string>
      </property>
     </widget>
     <widget class="QPushButton" name="chooseImage_Btn">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>250</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>选择背景图片</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QPushButton" name="page_3_previous_Btn">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>上一步</string>
      </property>
     </widget>
     <widget class="QPushButton" name="page_3_next_Btn">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>475</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>下一步</string>
      </property>
     </widget>
     <widget class="QPushButton" name="page_3_setTitle_Btn">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>250</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>设置标题内容</string>
      </property>
     </widget>
     <widget class="QLabel" name="page_3_imageBackground_Label">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>630</width>
        <height>460</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="E_Card_Title_Label">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>630</width>
        <height>50</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="page_3_backgroundColor_Btn">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>175</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>设置标题背景</string>
      </property>
     </widget>
     <widget class="QPushButton" name="page_3_fontStyle_Btn">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>325</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>设置字体格式</string>
      </property>
     </widget>
     <widget class="QPushButton" name="page_3_fontColor_Btn">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>400</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>设置字体颜色</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QPushButton" name="page_4_previous_Btn">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>上一页</string>
      </property>
     </widget>
     <widget class="QPushButton" name="page_4_cardInfoFont_Btn">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>200</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>设置贺卡字体</string>
      </property>
     </widget>
     <widget class="QLabel" name="page_4_title_Label">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>630</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="page_4_imageBackground_Label">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>630</width>
        <height>460</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="page_4_cartInfo_Text">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>630</width>
        <height>460</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="page_4_cartInfoColor_Btn">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>300</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>设置字体颜色</string>
      </property>
     </widget>
     <widget class="QPushButton" name="page_4_save_Btn">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>400</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="music_start_Btn">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>100</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>播放</string>
    </property>
   </widget>
   <widget class="QSlider" name="music_Playing_HSlider">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>200</y>
      <width>271</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="music_volumn_VSlider">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>50</y>
      <width>22</width>
      <height>111</height>
     </rect>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="music_addMusic_Btn">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>140</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>本地音乐</string>
    </property>
   </widget>
   <widget class="QListWidget" name="music_musicList_ListWidget">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>240</y>
      <width>291</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="music_previous_Btn">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>100</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>上一首</string>
    </property>
   </widget>
   <widget class="QPushButton" name="music_next_Btn">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>100</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>下一首</string>
    </property>
   </widget>
   <widget class="QPushButton" name="music_volumn_Btn">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>140</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>音量</string>
    </property>
   </widget>
   <widget class="QLabel" name="music_playTime_Label">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>200</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="music_allTime_Label">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>200</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
